src/workloads/scale/LargeScaleModel.yml

```diff
@@ -337,8 +337,3 @@ Actors:
   - ExternalPhaseConfig:
       Path: ../../phases/scale/LargeScalePhases.yml
       Key: OplogTrailerCmd
-
-AutoRun:
-  Requires:
-    mongodb_setup:
-    - standalone
```

Comment:
This workload generates BFs sometimes. It's not doing anything useful so let's just disable it for now.